test/users.js

```diff
@@ -0,0 +1,156 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  User = require('../app/models').User;
+
+chai.should();
+
+describe('/users POST', () => {
+  it('should pass sign up, parameters are valid', () => {
+    return chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'Rodrigo',
+        lastName: 'Aparicio',
+        email: 'rodrigo.aparicio@wolox.com.ar',
+        password: '12345678a'
+      })
+      .then(res => {
+        res.should.have.status(200);
+        dictum.chai(res, 'a new user is created with the parameters sent');
+      });
+  });
+
+  it('should fail sign up, password is invalid has no numbers', () => {
+    return chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'Rodrigo',
+        lastName: 'Aparicio',
+        email: 'rodrigo.aparicio@wolox.com.ar',
+        password: 'helloworld'
+      })
+      .catch(err => {
+        err.should.have.status(400);
+        err.response.should.be.json;
+        err.response.body.should.have.property('message');
+        err.response.body.should.have.property('internal_code');
```

Comment:
check for your specific error, in some cases you may recieve a 400 for a different reason from what you were expecting and the test will pass even if it wasnt correct.